spec/lib/tasks/dupicate_rent_periods_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+require "rake"
+
+RSpec.describe "duplicate_rent_periods" do
+  before do
+    Rake.application.rake_require("tasks/duplicate_rent_periods")
+    Rake::Task.define_task(:environment)
+    allow(Rails.logger).to receive(:info)
+    organisation = create(:organisation, rent_periods: (1..11).to_a)
+
+    [2, 4, 6, 11].each do |rent_period|
```

Comment:
I don't know if it shows up on live, but to me it seems worth testing with at least one group of more than 2 duplicates.